lib/DDG/Spice/BikeSharing/CitiBikeNYC.pm

```diff
@@ -10,6 +10,12 @@ spice wrap_jsonp_callback => 1;
 spice is_cached => 0;
 spice proxy_cache_valid => '200 304 15m';
 
+spice alt_to => {
+	status => {
```

Comment:
Done... I was actually understanding how things worked.. but was being p0wned by the fact that you need to add a trailing `/` to the URL for the NGINX to apply the rewrite rule :(